<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadIVUS</class>
 <widget class="QWidget" name="ReadIVUS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>打开图片</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>左侧层数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_l"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>右侧角度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_r">
       <property name="maximum">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>截取图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QVTKOpenGLWidget" name="transverse" native="true">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(191, 64, 64, 0);
background-image: url(:/Style/StyleResources/branch_close.png);
}
QPushButton:hover{
background-color:rgb(50,50,50);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QVTKOpenGLWidget" name="median_sagittal" native="true">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>512</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>512</height>
        </size>
       </property>
       <widget class="QSlider" name="slider">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>16</width>
          <height>512</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>512</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>512</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
